vibes-tools/tools/vibes-constants/lib/asm.mli

```diff
@@ -1,5 +1,4 @@
 (** Predefined labels for use with the system assembler. *)
 
-val relative_patch_placement : string
```

Comment:
We don' need this anymore?

Reply:
I didn't see it being used anywhere. Was there a reason for adding it?

Reply:
There probably was a reason, but if it's gone, good riddance. I bet it was used to calculate jumps in the assembler, but there could easily be a different method that's better